challenges/letter-count.js

```diff
@@ -19,15 +19,30 @@
   - Instead of just counting letters, calculate their percent-based frequency.
     See: http://www.math.cornell.edu/~mec/2003-2004/cryptography/subs/frequencies.html
 
-    ```javascript
-    {
-      "a": 0.2, // percent
-      "p": 0.4,
-      "l": 0.2,
-      "e": 0.2
-    }
-    ```
-
 */
 
 // YOUR CODE HERE
+Object.size = function letterCount (obj) {
```

Comment:
Modifying a built-in data type is generally frowned upon. But I like that you're moving in an OO direction.